build.psm1

```diff
@@ -2344,28 +2382,80 @@ function New-AppxPackage
     $appxManifest = $appxManifest.Replace('#SQUARE150x150LOGO#', 'Assets\Powershell_256.png')
     $appxManifest = $appxManifest.Replace('#SQUARE44x44LOGO#', 'Assets\Powershell_48.png')
 
-    Write-Verbose "Place Appx Manifest in $SourcePath"
-    $appxManifest | Out-File "$SourcePath\AppxManifest.xml" -Force
+    Write-Verbose "Place Appx Manifest in $PackageSourcePath"
+    $appxManifest | Out-File "$PackageSourcePath\AppxManifest.xml" -Force
 
-    $assetsInSourcePath = Join-Path $SourcePath 'Assets'
+    $assetsInSourcePath = Join-Path $PackageSourcePath 'Assets'
     New-Item $assetsInSourcePath -type directory -Force | Out-Null
 
     Write-Verbose "Place AppxManifest dependencies such as images to $assetsInSourcePath"
     Copy-Item "$AssetsPath\*.png" $assetsInSourcePath -Force
 
-    $appxPackageName = $PackageName + "_" + $PackageVersion
+    $appxPackageName = $PackageName + "_" + $PackageSemanticVersion
     if ($PackageNameSuffix) {
         $appxPackageName = $appxPackageName, $PackageNameSuffix -join "-"
     }
-    $appxPackagePath = "$pwd\$appxPackageName.appx"
-    Write-Verbose "Calling MakeAppx from $makeappxExePath to create the package @ $appxPackagePath"
-    & $makeappxExePath pack /o /v /d $SourcePath  /p $appxPackagePath | Write-Verbose
+    $appxLocationPath = "$pwd\$appxPackageName.appx"
+    Write-Verbose "Calling MakeAppx from $makeappxExePath to create the package @ $appxLocationPath"
+    & $makeappxExePath pack /o /v /d $PackageSourcePath  /p $appxLocationPath | Write-Verbose
 
     Write-Verbose "Clean-up Appx artifacts and Assets from $SourcePath"
     Remove-Item $assetsInSourcePath -Recurse -Force -ErrorAction SilentlyContinue
-    Remove-Item "$SourcePath\AppxManifest.xml" -Force -ErrorAction SilentlyContinue
+    Remove-Item "$PackageSourcePath\AppxManifest.xml" -Force -ErrorAction SilentlyContinue
+
+    Write-Verbose "You can find the APPX @ $appxLocationPath" -Verbose
+    $appxLocationPath
+}
+
+# Function to create a zip file for Nano Server and xcopy deployment
+function New-ZipPackage
+{
+    [CmdletBinding()]
+    param (
+    
+        # Name of the Product
+        [ValidateNotNullOrEmpty()]
+        [string] $PackageName = 'PowerShell',
+
+        # Suffix of the Name
+        [string] $PackageNameSuffix,
 
-    return $appxPackagePath
+        # Version of the Product
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $PackageVersion,
+
+        # Source Path to the Product Files - required to package the contents into an Zip
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $PackageSourcePath
+    )
+
+    $ProductSemanticVersion = Get-PackageSemanticVersion -Version $PackageVersion
+
+    $PackageVersion = Get-PackageVersionAsMajorMinorBuildRevision -Version $PackageVersion
```

Comment:
Not sure of the ask. Do you want me to remove the variable or call to write-verbose or something else?